finder-app/writer.c

```diff
@@ -0,0 +1,47 @@
+/****************************************************************************************
+@Author 	Harsh Beriwal (harsh.beriwal@colorado.edu)
+
+@course 	Advanced Embedded Software Development
+@instructor	Daniel Walkes 
+****************************************************************************************/
+
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <syslog.h> 
+#include <stdlib.h>
+#include <string.h> 
+
+int main (int argc, char *argv[])
+{
+	int fd; 
+	ssize_t wr_fl; 
+	openlog(NULL, 0 , LOG_USER);     
+	char *writefile = argv[1];					 //Storing Arguments in the Writefile and WriteStr
+	char *writestr = argv[2];
```

Comment:
Good idea to save to variables - look at const correct discussion from class today for an additional suggestion.